src/store/reducers/stateCache/stateCache.ts

```diff
@@ -66,49 +66,46 @@ const stateCacheSlice = createSlice({
     setToastMsg: (state, action: PayloadAction<ToastMsgType | undefined>) => {
       state.toastMsg = action.payload;
     },
-    setPauseQueueConnectionCredentialRequest: (
-      state,
-      action: PayloadAction<boolean>
-    ) => {
-      state.queueConnectionCredentialRequest = {
-        ...state.queueConnectionCredentialRequest,
+    setPauseQueueIncomingRequest: (state, action: PayloadAction<boolean>) => {
+      state.queueIncomingRequest = {
+        ...state.queueIncomingRequest,
         isPaused: action.payload,
```

Comment:
looks like `isPaused` and `isProcessing` are the opposite. We could use just one variable.